FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/Common/http_demo_utils.c

```diff
@@ -118,7 +118,13 @@ BaseType_t connectToServerWithBackoffRetries( TransportConnect_t connectFunction
             LogInfo( ( "Retry attempt %lu out of maximum retry attempts %lu.",
                        ( xReconnectParams.attemptsDone + 1 ),
                        RETRY_MAX_ATTEMPTS ) );
-            xBackoffAlgStatus = BackoffAlgorithm_GetNextBackoff( &xReconnectParams, &usNextBackoff );
+
+            /* Generate a random number and calculate backoff value (in milliseconds) for
+             * the next connection retry.
+             * Note: It is recommended to seed the random number generator with a device-specific
+             * entropy source so that possibility of multiple devices retrying failed network operations
+             * at similar intervals can be avoided. */
+            xBackoffAlgStatus = BackoffAlgorithm_GetNextBackoff( &xReconnectParams, uxRand(), &usNextBackoff );
```

Comment:
`BackoffAlgorithm_InitializeParams` not updated with correct parameters here.

Reply:
Updated